util/topgen.py

```diff
@@ -583,29 +590,26 @@ def generate_racl(topcfg: Dict[str, object], out_path: Path) -> None:
     log.info('Generating RACL Control IP with ipgen')
     topname = topcfg['name']
 
-    for racl_group, policies in topcfg['racl']['policies'].items():
-        params = {
-            "nr_role_bits": 4,
-            "nr_ctn_uid_bits": 8,
-            "nr_policies": len(policies),
-            "policies": policies
-        }
-
-        # If we have more RACL policy groups, uniquify the control IP
-        # if len(topcfg['racl']['policies']) > 1:
-        #     params['module_instance_name'] = f'racl_ctrl_{racl_group}'
-
-        # # Only render the RACL groups that are really instantiated in that top
-        # for m in topcfg['module']:
-        #     if m['name'] == params['module_instance_name']:
-        #         ipgen_render("racl_ctrl", topname, params, out_path)
-        #         break
-        # TODO(#25673): The obove code, would be the correct if ipgen correctly supports rendering
```

Comment:
Would you like this TODO with attached issue to remain in the code somewhere? (just wondering; no change needed)

Reply:
This TODO is implicitly solved by this PR. If there are multiple RACL groups, it means that there are uniquified `racl_ctrl` instances, which was not possible before.

Reply:
Sounds good to me.

Reply:
You are right ;-) Though I haven't fully decided on how to make the mental connection between the uniquified IP name and the RACL policy group. I am fully aware of that, and it will be solved properly in a follow-up PR.